packages/testkit/package.json

```diff
@@ -17,13 +27,14 @@
   },
   "scripts": {
     "test:e2e": "npx playwright test",
-    "build": "tsc",
+    "build": "tsc && tsc --project tsconfig.esm.json",
     "start-server": "yarn lerna run storybook --scope=monday-ui-react-core"
   },
   "bugs": {
     "url": "https://github.com/mondaycom/vibe/issues"
   },
   "devDependencies": {
-    "@playwright/test": "1.45.3"
+    "@playwright/test": "1.45.3",
+    "typescript": "^4.4.3"
```

Comment:
any reason for not using a newer version?

Reply:
used the same version as in the root project. I don't see a reason why not to use a newer version. @talkor do you?

Reply:
@YossiSaadi yes it should work this way, but that means that when you'll upgrade the TS version you'll have to deal with all packages in one go. If you want, you CAN set each package using its own TS version. This will allow you a more flexible approach

Reply:
yep, exactly
we do want to have the root with packages that should have the same version across all the repos, but were not there yet
eslint, typescript, prettier, etc.

but I think for now, @uziab can use the one from root (or remove the declaration entirely, up to him to decide, not critical for me)

Reply:
removed typescript